ui/packages/consul-ui/app/components/action/index.hbs

```diff
@@ -1,35 +1,27 @@
-{{#if @for}}
+{{#if @for~}}
```

Comment:
What does the `~` do here?

Reply:
It's to signal that there should be no whitespace in the produced output between this and the next `~`.

In this particular case I think it was down to me seeing output like this:

```hbs

<Action>link</Action>.
---
<a href="">link</a> .
```

Note the space after the `<a>` but before the period, which handlebars/ember was adding due to the spacing in the component.